tests/acceptance/features/apiSearch/contentSearch.feature

```diff
@@ -161,3 +162,32 @@ Feature: content search
     Examples:
       | dav-path-version |
       | spaces           |
+
+
+  Scenario Outline: search across files with different format
+    Given using <dav-path-version> DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the GraphApi
+    And user "Alice" has uploaded file with content "this is a simple text file" to "test-text-file.txt"
+    And user "Alice" has uploaded file with content "this is a simple pdf file" to "test-pdf-file.pdf"
+    And user "Alice" has uploaded file with content "this is a simple cpp file" to "test-cpp-file.cpp"
+    And user "Alice" has uploaded file with content "this is another text file" to "testfile.txt"
+    And user "Alice" has uploaded file "filesForUpload/testavatar.png" to "/testavatar.png"
+    And user "Alice" has uploaded a file inside space "project-space" with content "this is a simple markdown file" to "test-md-file.md"
+    And user "Alice" has uploaded a file inside space "project-space" with content "this is a simple odt file" to "test-odt-file.odt"
+    When user "Alice" searches for "Content:simple" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain these entries:
+      | test-text-file.txt |
+      | test-pdf-file.pdf  |
+      | test-cpp-file.cpp  |
+      | test-md-file.md    |
+      | test-odt-file.odt  |
+    But the search result of user "Alice" should not contain these entries:
+      | testavatar.png |
+      | testfile.txt   |
```

Comment:
```feature
    And the search result of user "Alice" should contain only these files:
      | test-text-file.txt |
      | test-pdf-file.pdf  |
      | test-cpp-file.cpp  |
      | test-md-file.md    |
      | test-odt-file.odt  |
```

In place of this step Is this step appropriate?  since it will check `contains only` files and it will fail if extra unwelcome files are included on the search list. Also, the number of files is also checked in this step.

Reply:
sure I can update that.